clients/admin-ui/src/features/common/nav/v2/NavSideBar.tsx

```diff
@@ -25,11 +27,17 @@ export const NavSideBar = () => {
           // We still need to handle cross-zone links.
           const { href, isActive } = resolveZoneLink({ href: path, router });
 
-          return (
-            <NavSideBarLink key={title} href={href} isActive={isActive}>
-              {title}
-            </NavSideBarLink>
-          );
+          if (
+            href === "/privacy-requests/configure" &&
+            !features.flags.privacyRequestsConfiguration
```

Comment:
This toggling of certain routes based on feature flags should be done from the nav config helpers themselves instead of within component markup. Check out [configureNavGroups](https://github.com/ethyca/fides/blob/37c18c598b5d04e6382f098b8cf6ba87073800ad/clients/admin-ui/src/features/common/nav/v2/nav-config.ts#L78) and its unit tests -- it already takes in some features to decide what should be shown.